spring-xd-dirt/src/test/java/org/springframework/xd/dirt/integration/bus/kafka/KafkaMessageBusTests.java

```diff
@@ -33,13 +35,14 @@
 import java.util.concurrent.TimeUnit;
 
 import kafka.api.OffsetRequest;
+
 import org.apache.kafka.clients.producer.KafkaProducer;
 import org.apache.kafka.clients.producer.ProducerConfig;
-import org.hamcrest.CoreMatchers;
+import org.hamcrest.collection.IsMapContaining;
 import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
-
+import org.springframework.amqp.utils.test.TestUtils;
```

Comment:
We should probably pull in the integration version of this class rather than amqp - will fix on merge